### PR TITLE
DATAGO-5880: Learn content is updated to explain accounts and how cus…

### PR DESCRIPTION
**Pull request by andreygorelov**
…tomers can leverage them

### CODE REVIEW

File: docs/group_faqs/gf_multiple_account.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Solace Cloud Multiple Account
+=============================
+
+Overview
+~~~~~~~~
+Users can be added to more than one Solace Cloud Account. Once added, a user can have different access and permission in each environment.
```
Comment (macstewart) [inline docs/group_faqs/gf_multiple_account.rst]: 'permission' should be 'permissions'. Also, access and permissions mean the same thing in this context so maybe reword it to be less redundant?

File: docs/group_faqs/gf_multiple_account.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+For example, an organization may purchase two different subscriptions for (production, non-production, etc.) environments. The organization will be able to add users to one or more environments, enabling them to segregate their user base e.g. contractors, sales engineers, and different profession service groups.
```
Comment (macstewart) [inline docs/group_faqs/gf_multiple_account.rst]: I don't understand how an organization purchasing different subscriptions has anything to do with a user having different access/permissions in different accounts. By leading with 'for example' you're saying that it's an example for your previous point.

File: docs/group_faqs/gf_multiple_account.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Access and Permissions
+~~~~~~~~~~~~~~~~~~~~~~
+The Multiple Account feature allows organizations to provide to their users’ different access and permissions to sets of different Solace Cloud PubSub+ Messaging Services. In addition, users can have different roles (e.g. admin, billing admin, and messaging user) in separate accounts.
```
Comment (macstewart) [inline docs/group_faqs/gf_multiple_account.rst]: 'users' should not have an apostrophe in this context.

File: docs/group_faqs/gf_multiple_account.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+User with multiple account
```
Comment (macstewart) [inline docs/group_faqs/gf_multiple_account.rst]: I don't think this header is useful

File: docs/group_faqs/gf_multiple_account.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Login
+~~~~~
+The same credentials are used to login to all accounts in Solace PubSub+ Cloud Console. After successful login, a user that belongs to more than one account is prompted to select which account to be login to. If the user belongs to only one account, it will be directed to the console.
```
Comment (macstewart) [inline docs/group_faqs/gf_multiple_account.rst]: I'm not sure the first sentence is needed, as the second sentence already makes it clear that the account selection is done after log in. 

"select which account to be login to" should be "select which account to log in to" grammatically, and I think "select which account to access" sounds better. 

"If the user belongs to only one account, it will be directed to the console." is unclear. Reword to something like "A user in only one account will not be prompted and will be signed in immediately."

**Comment by johnvincentcorpuz** [comment]
FYI for the reviewers.. For PRs, there's a temporary site that gets deployed: http://pr68.maas-docs-online.k8s.mymaas.net/ to preview your changes. Gets torn down to when the PR is closed.

So pr<##>.maas-docs-online.k8s.mymaas.net
